data/landSales/getLandSales.ts

```diff
@@ -0,0 +1,272 @@
+  name: string;
+
+}
+
+type LandGroup = {
+  x: number;
+  y: number;
+  numLands: number;
+  reserved: string;
+  originalX: number;
+  originalY: number;
+  bundleId?: string;
+}
+
+async function generateLandsForMerkleTree(sectorData: SectorData, bundles: {[id: string]: string[]}, prices: {[priceId: string]: string}, log?: boolean): Promise<{lands: SaleLandInfo[], partnersLands: PartnerLandInfo[]}> {
+  const partnersLands: PartnerLandInfo[] = [];
+  const lands: SaleLandInfo[] = [];
+  let numLands = 0;
+  let numLandsInInput = 0;
+  let num1x1Lands = 0;
+  let num3x3Lands = 0;
+  let num6x6Lands = 0;
+  let num12x12Lands = 0;
+  let num24x24Lands = 0;
+  let numSandboxReservedGroups = 0;
+  let numSandboxReserved = 0;
+  let numReserved = 0;
+  let numReservedGroup = 0;
+  let numBundles = 0;
+
+  function addLandGroup(landGroup: LandGroup) {
+    const size = Math.sqrt(landGroup.numLands);
+    if (size * size !== landGroup.numLands) {
+      reportError("wrong number of land " + landGroup.numLands);
+    }
+    let assetIds: string[] = [];
+    if (landGroup.bundleId) {
+      assetIds = bundles[landGroup.bundleId];
+      numBundles++;
+    }
+    if (!assetIds) {
+      throw new Error("assetIds cannot be undefined");
+    }
+
+    const premium = assetIds.length > 0;
+    let priceId = '';
+    if (size === 1) {
+      num1x1Lands++;
+      priceId = (premium ? "premium_" : "") + "1x1";
+    } else if (size === 3) {
+      num3x3Lands++;
+      priceId = (premium ? "premium_" : "") + "3x3";
+    } else if (size === 6) {
+      num6x6Lands++;
+      priceId = (premium ? "premium_" : "") + "6x6";
+    } else if (size === 12) {
+      num12x12Lands++;
+      priceId = (premium ? "premium_" : "") + "12x12";
+    } else if (size === 24) {
+      num24x24Lands++;
+      priceId = (premium ? "premium_" : "") + "24x24";
+    } else {
+      reportError("wrong size : " + size);
+    }
+    const price = prices[priceId];
+    if (!price) {
+      reportError("no price for size = " + size);
+    }
+
+    if (!(landGroup.x % size === 0 && landGroup.y % size === 0)) {
+      reportError(
+        "invalid coordinates: " +
+          JSON.stringify({
+            x: landGroup.x,
+            originalX: landGroup.originalX,
+            y: landGroup.y,
+            originalY: landGroup.originalY,
+            size,
+          })
+      );
+      return;
+    }
+
+    if (landGroup.x < 0 || landGroup.x >= 408) {
+      reportError("wrong x : " + landGroup.x);
+      return;
+    }
+    if (landGroup.y < 0 || landGroup.y >= 408) {
+      reportError("wrong y : " + landGroup.y);
+      return;
+    }
+
+    if (landGroup.reserved) {
+      numReservedGroup++;
+      numReserved += size * size;
+      if (landGroup.reserved.toLowerCase() === sandboxWallet.toLowerCase()) {
+        numSandboxReservedGroups++;
+        numSandboxReserved += size * size;
+      }
+      const name = "unknown : " + landGroup.reserved;
+      // switch (land.reserved) {
```

Comment:
Why commented out?

Reply:
that was copied over. these check were there when there was name for each reserved address and checks were made against them